Documentation/Specification/silica.tex

```diff
@@ -1749,7 +1749,7 @@
 Returns the mirror of the \term{sheet} \arg{sheet}.  If the sheet is not a
 subclass of \cl{mirrored-sheet-mixin}, this will return \cl{nil}.  If the sheet
 is a subclass of \cl{mirrored-sheet-mixin} and does not currently have a mirror,
-\cl{sheet-mirror} will return \cl{nil}.
+\cl{sheet-direct-mirror} will return \cl{nil}.
 
```

Comment:
technically it is not wrong statement, but I get that given the context it may be confusing.